corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -186,7 +186,7 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
 
         this.setQueryData = function (inputs, execute, forceManualSearch) {
             var selections = hqImport("cloudcare/js/formplayer/utils/util").currentUrlToObject().selections;
-            this.queryData ||= {};
+            this.queryData = queryData || {};
```

Comment:
should this be `this.queryData = this.queryData || {};`

Reply:
yes 5429f16efd0